org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/AbstractMatcher.java

```diff
@@ -129,6 +129,16 @@ final void nextIsSwitch() {
 		}
 	}
 
+	final void nextIsLabel() {
```

Comment:
I mean it is kinda obvious what this function does looking at the other ones as well. However, A little comment would be nice. 
```suggestion
         /**
	 * Moves {@link #cursor} to next instruction if it is
	 * <code>LABEL</code>, otherwise sets it to <code>null</code>.
	 */
	final void nextIsLabel() {
```